lib/Requestable.js

```diff
@@ -256,9 +256,17 @@ class Requestable {
             results.push(...thisGroup);
 
             const nextUrl = getNextPage(response.headers.link);
-            if (nextUrl && !(options && typeof options.page !== 'number')) {
-               log(`getting next page: ${nextUrl}`);
-               return this._requestAllPages(nextUrl, options, cb, results);
+            if(nextUrl) {
+               options.page = parseInt(
```

Comment:
It's possible for options to be undefined. Please handle this prior to setting page.

Reply:
ok will do that